testing/tsconfig.json

```diff
@@ -5,6 +5,6 @@
     "types": ["node"],
     "skipLibCheck": true
   },
-  "include": ["**/*.ts", "**/*.tsx"],
-  "exclude": ["node_modules", "dist"]
+  "include": ["playwright/**/*.ts", "playwright/**/*.tsx", "*.ts"],
+  "exclude": ["node_modules", "dist", "cypress", "**/*.cy.ts"]
```

Comment:
why are we excluding "cypress" and "cy" file extensions when we are no longer using it?